chain/network/src/test_utils.rs

```diff
@@ -149,29 +149,28 @@ pub fn random_epoch_id() -> EpochId {
 
 // Compare whenever routing table match.
 pub fn expected_routing_tables(
-    current: HashMap<PeerId, Vec<PeerId>>,
-    expected: Vec<(PeerId, Vec<PeerId>)>,
+    got: &HashMap<PeerId, Vec<PeerId>>,
+    want: &Vec<(PeerId, Vec<PeerId>)>,
```

Comment:
Thanks, going to steal got/want names for myself, much better than expected/actual
```suggestion
    got: &HashMap<PeerId, Vec<PeerId>>,
    want: &[(PeerId, Vec<PeerId>)],
```